### PR TITLE
feat(S1/S1): Add support for `action.type` 25,47,5232

### PR DESCRIPTION
**Pull request by PierrickV**
This event types are currently not parsed because of missing fields in taxonomy files

With this new capabilities, new Smart Descriptions are added https://github.com/SEKOIA-IO/Community/pull/96

### CODE REVIEW

File: SentinelOne/sentinelone/tests/activity-type-5232.json [hunk 1]
```diff
@@ -35,7 +39,12 @@
       "id": "551799238352448315"
     },
     "rule": {
-      "id": "556166862007673241"
+      "description": "Flux",
+      "id": "556166862007673241",
+      "name": "Block all"
```
Comment (Darkheir) [inline SentinelOne/sentinelone/tests/activity-type-5232.json:44]: Strange, no ?

Comment (PierrickV) [reply SentinelOne/sentinelone/tests/activity-type-5232.json:44]: What is strange ?

It comes from the following fields:
```json
    "ruleDescription": "Flux",
    "ruleId": 556166862007673241,
    "ruleName": "Block all",
```

**Review by Darkheir — APPROVED** [review]

File: SentinelOne/sentinelone/tests/activity-type-5232.json (hunk 1 above)
Comment (Darkheir) [reply SentinelOne/sentinelone/tests/activity-type-5232.json:44]: Fair enough, I just find strange to have only "Flux" in description and "Block All" as the rulename

Comment (PierrickV) [reply SentinelOne/sentinelone/tests/activity-type-5232.json:44]: :shrug:

**Pull request merged**